EmpireLegends/BiddingFacility.cpp

```diff
@@ -25,32 +25,48 @@ int BiddingFacility::getPlayerBid(Player* player)
 	return bids[player];
 }
 
-void BiddingFacility::placeBids(vector<Player*> biddingPlayers) {
+ string BiddingFacility::placeBids(vector<Player*> biddingPlayers) {
 
 	auto playerBid = 0;
 	string temp;
 
 	// each players places their bid
-	for (auto& player : biddingPlayers)
-	{
-		do {
-			std::cout << "\n" << player->getName() << " place your bid: ";
-			std::cin >> playerBid;
-		} while (playerBid > player->getCoins() || playerBid < 0);
-
-		system("cls");
+	
+		for (auto& player : biddingPlayers)
+		{
+			//For Bot Players
+			int bot = player->getName().find("Bot");
```

Comment:
i think it would be more intuitive if this was `bool isBot = player->getName().find("Bot") !== std::string::npos`, which would then make understanding the condition of your "if/else" loop easier